### PR TITLE
Add example for a custom cast receiver app

### PR DESCRIPTION
**Pull request by stonko1994**
## Description
This PR will add an example of what is needed for a custom Cast Receiver app.

### CODE REVIEW

File: castReceiver/index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    <div class="container">
+        <h1>HTML5 Adaptive Streaming Player for MPEG-DASH & HLS</h1>
+        <h2>Your videos play everywhere with low startup delay, no buffering and in highest quality.</h2>
+        <div class="content">
+            <div class="player-wrapper">
+                <div id="player"></div>
+            </div>
+            <div class="description">
+                <p>For more information about the bitmovin player, please have a look at our online <a
+                        href="//bitmovin.com/support" target="_blank">Developer Section</a>.</p>
```
Comment (jmsn) [inline castReceiver/index.html]: Looks like the location https://bitmovin.com/support has changed. It now points to open positions in the Support department 😄.

File: castReceiver/index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+<html lang="en">
+<head>
+    <title>Bitmovin Cast Receiver Demo</title>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400" rel="stylesheet">
+    <link rel="icon" type="image/png" href="../images/bit-fav.png">
+    <!-- Bitmovin Player -->
+    <script src="//cdn.bitmovin.com/player/web/8.5.0/bitmovinplayer.js"></script>
```
Comment (jmsn) [inline castReceiver/index.html]: As on the receiver we're using the latest version 8 player version we probably want to use it here too

File: castReceiver/index.html (hunk 1 above)
Comment (stonko1994) [reply castReceiver/index.html]: We know, was already reported 😄

File: castReceiver/index.html (hunk 2 above)
Comment (stonko1994) [reply castReceiver/index.html]: AFAIK we have some issues if the versions are not matching. For example I tried player `8.5.0` with the `8.0.0/bitmovinplayer-remotereceiver.js` and nothing worked (also no error message was shown). And we don't have a `8.latest` link to the `bitmovinplayer-remotereceiver.js ` yet so I explicitly decided to use specific versions here. But sure we could use `8.6.0`

**Comment by stonko1994** [comment]
@jmsn We don't have such instructions in other examples 🤔

File: castReceiver/receiverApp.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    // Instantiate a Bitmovin player
+    const player = new bitmovin.player.Player(document.getElementById('player-wrapper'), conf);
+
+    // Hide loading overlay when a source is loaded
+    player.on(bitmovin.player.PlayerEvent.SourceLoaded, hideStatusOverlay);
+
+    // Hide overlay when an error happens (avoids hidden errors during startup)
+    player.on(bitmovin.player.PlayerEvent.Error, hideStatusOverlay);
+
+    // Handler that takes properties from the remotecontrol.customReceiverConfig configuration for use within the receiver
```
Comment (jmsn) [inline castReceiver/receiverApp.html]: Sorry forgot one thing: This comment is slightly misplaced, it belongs to line 127.

Side note:
The `GoogleCastRemoteControlReceiver` is our actual receiver logic that uses the google cast framework and handles the communication with the sender, etc. Its creation does not directly relate to the comment.

**Pull request merged**